build/elasticsearch/bin/es-docker

```diff
@@ -22,4 +22,6 @@ do
     fi
 done < <(env)
 
+echo "-Des.cgroups.hierarchy.override=/" >> config/jvm.options
```

Comment:
@dliappis Given your thoughts here, which I agree with, I wonder what you think of:

```diff
diff --git a/build/elasticsearch/bin/es-docker b/build/elasticsearch/bin/es-docker
index b50046e..de544fd 100755
--- a/build/elasticsearch/bin/es-docker
+++ b/build/elasticsearch/bin/es-docker
@@ -22,6 +22,6 @@ do
     fi
 done < <(env)
 
-echo "-Des.cgroups.hierarchy.override=/" >> config/jvm.options
+export ES_JAVA_OPTS="-Des.cgroups.hierarchy.override=/ $ES_JAVA_OPTS"
 
 exec bin/elasticsearch ${es_opts}
```